ios/Integreat/Integreat.entitlements

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>aps-environment</key>
```

Comment:
Do you know what this means? Why is development mentioned here?

Reply:
it's an auto-generated file, Xcode sets the value of the Apple Push Notification environment entitlement based on the app's current provisioning profile. I expect it will also be generated when you build the app on your Xcode. Would be ok to delete from my side or keep it.